src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.ts

```diff
@@ -37,6 +38,8 @@ export class OffsetablePieceComponent implements OnChanges, OnDestroy {
 
   public media?: Media
 
+  public positionInVideoInMs: number
```

Comment:
It would be nice with a default value of `0`.

Reply:
Updated